### DISCUSSION TITLE
add algorithm param to `/info` api

### DISCUSSION THREAD

**Discussion by JinIgarashi** [developmentseed/titiler#785] [Feature Request]
In titiler, `/info` and `/statistics` api provide different information. While `/statistics` api has parameters for `algorithm` and `algorithm_params`, `/info` api does not have them.
After applying an algorithm, the characteristic of the raster will be significantly changed, it would be nice if `/info` api also has algorithm parameters.

**Comment by vincentsarago** [comment]
the `/info` endpoint should return the metadata from the data itself, not the transformed data. I added `processing` feature to the `/statistics` endpoint because it return value from image so IMO it made sense to allow image processing there. For the `/info` endpoint we only read the metadata of the dataset so I'm not sure how/where we could apply algorithm there 🤷

**Reply by JinIgarashi** [reply]
I see your point. maybe it is difficult to apply algorithm to `/info` endpoint.
